tests/jepsen/src/memgraph/query.clj

```diff
@@ -37,15 +35,16 @@
   RETURN n.id as id;
   ")
 
-(dbclient/defquery max-id
-  "MATCH (n:Node)
-  RETURN max(n.id) as id;
-  ")
-
 (defn add-nodes
-  [start-idx end-idx]
+  [batch-size]
   (dbclient/create-query
-   (str "FOREACH (i in range(" start-idx ", " end-idx ") | CREATE (:Node {id: i}));")))
+   (let [query
+         (str "MATCH (n:Node) WITH coalesce(max(n.id), 0) as max_idx "
+              "FOREACH (i in range(max_idx+1, max_idx+"
+              batch-size
+              ") | CREATE (:Node {id: i}));")]
```

Comment:
not sure how this driver works, but usually there is a way to pass in parameters.
instead of generating the string, you could just have a constant string and pass in the batch-size as a parameter.
all okay if this is not possible